scripts/xpcs_online_boost_client.py

```diff
@@ -53,6 +53,9 @@ def arg_parse():
 
     #Processing type
     atype = args.atype
+    #edge case where DM does not send a type on the call 
+    if not atype:
+        atype = 'Both'
```

Comment:
`args.atype` already defaults to 'Both' above in argparse. Per @AZjk, we should change this to default to 'Multitau' if no option is given. 

We should probably remove 'Both' entirely for now, since that's running multitau first then overwriting the same dataset with twotime.